python/requirements/compat/requirements_legacy_compat.txt

```diff
@@ -21,8 +21,8 @@ torchvision==0.10.0
 pytorch-lightning==1.5.10
 
 # Upstream libraries
-lightgbm_ray==0.1.5
-xgboost_ray==0.1.10
+xgboost_ray==0.1.18
+lightgbm_ray==0.1.9
```

Comment:
Lower and upper bound need to be the latest, otherwise the tests will fail. Is this ok @krfricke?